src/libImaging/Unpack.c

```diff
@@ -413,24 +413,14 @@ unpackL4IR(UINT8 *out, const UINT8 *in, int pixels) {
 }
 
 static void
-unpackLA(UINT8 *_out, const UINT8 *in, int pixels) {
-    int i;
-    /* LA, pixel interleaved */
-    for (i = 0; i < pixels; i++) {
-        UINT32 iv = MAKE_UINT32(in[0], in[0], in[0], in[1]);
-        memcpy(_out, &iv, sizeof(iv));
-        in += 2;
-        _out += 4;
-    }
-}
-
-static void
-unpackLAL(UINT8 *_out, const UINT8 *in, int pixels) {
+unpackLAL(UINT8 *out, const UINT8 *in, int pixels) {
     int i;
+    UINT8 *in1 = in;
+    UINT8 *in2 = in + pixels;
     /* LA, line interleaved */
-    for (i = 0; i < pixels; i++, _out += 4) {
-        UINT32 iv = MAKE_UINT32(in[i], in[i], in[i], in[i + pixels]);
-        memcpy(_out, &iv, sizeof(iv));
+    for (i = 0; i < pixels; ++i) {
```

Comment:
```suggestion
    for (i = 0; i < pixels; i++) {
```
Just to be consistent with the other loops.